less/home-page.less

```diff
@@ -18,14 +20,36 @@
     display: flex;
     flex-wrap: wrap;
     width: 100%;
+    @media @phone{
+      .sky{
+        display:none;
+      }//closes sky
+
+    }//closes media
+    @media @phone{
+      .launch{
+        display:none;
+      }//closes launch
+
+    }//closes media
+    @media @phone{
+        .blast{
+          width:95%;
+        }
+    }//closes media
+
+    
```

Comment:
one tip here would be to put the `.blast` and `.launch` rules inside one `@media @phone {    }` tag